RENotebookApi/RENotebookApi.fsproj

```diff
@@ -1,15 +1,22 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net472</TargetFrameworks>
```

Comment:
We avoided adding netstandard here as it's meant for a particular place - https://github.com/fsprojects/IfSharp but if it makes things easier can add a new target.